Python/Product/PythonTools/PythonTools/Intellisense/IntellisenseController.cs

```diff
@@ -716,67 +716,67 @@ private class ExpressionCompletionWalker : PythonWalker {
             // TODO: Take into account params arrays
             // TODO: need to parse and see if we have keyword arguments entered into the current signature yet
             PythonSignature sig = sigHelpSession.SelectedSignature as PythonSignature;
-            if (sig != null) {
-                var prevBuffer = sig.ApplicableToSpan.TextBuffer;
-                var textBuffer = _textView.TextBuffer;
+            if (sig == null) {
+                return;
+            }
 
-                var targetPt = _textView.BufferGraph.MapDownToFirstMatch(
-                    new SnapshotPoint(_textView.TextBuffer.CurrentSnapshot, position),
-                    PointTrackingMode.Positive,
-                    EditorExtensions.IsPythonContent,
-                    PositionAffinity.Successor
-                );
+            var prevBuffer = sig.ApplicableToSpan.TextBuffer;
+            var textBuffer = _textView.TextBuffer;
 
-                if (targetPt == null) {
-                    return;
-                }
-                var span = targetPt.Value.Snapshot.CreateTrackingSpan(targetPt.Value.Position, 0, SpanTrackingMode.EdgeInclusive);
+            var targetPt = _textView.BufferGraph.MapDownToFirstMatch(
+                new SnapshotPoint(_textView.TextBuffer.CurrentSnapshot, position),
```

Comment:
Actually, I'll just remove the unused variable. It wouldn't prevent a race against the buffer changing anyway, but I think we're on the UI thread here so it doesn't matter.